.github/ISSUE_TEMPLATE/release_a_new_version.md

```diff
@@ -16,6 +16,7 @@ labels: release
 - [ ] Cut a branch for the new release out of the `develop` branch, following the gitflow naming pattern `release/0.1.0`.
 - [ ] Update the code to use this version number.
 - [ ] Update the [CHANGELOG.md] file with the new version number and release notes.
+- [ ] Update the badges version in the [README.md].
```

Comment:
```suggestion
- [ ] Update the badge versions in the [docs/00 - Stellar Anchor Platform.md].
```

Reply:
Done in https://github.com/stellar/java-stellar-anchor-sdk/pull/587/commits/145381598915796fc8515f59496ab1a09705e6b8.